docs/troubleshooting.rst

```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+###############
+
+.. _troubleshooting:
+
+We are doing the best we can so you do not have to read this section.
+
+In case you do, please find the common problems you may have below.
+
+If you do not find your problem here, please file an issue or ask for
+help on `#storage <http://chat.mibbit.com/?server=irc.mozilla.org&channel=#storage>`_
```

Comment:
I would prefer a link to communication channels http://kinto.readthedocs.org/en/latest/contributing.html#communication-channels